core/server/utils/image-size-from-url.js

```diff
@@ -1,103 +0,0 @@
-// Supported formats of https://github.com/image-size/image-size:
```

Comment:
Did you use `git mv` to rename the file?
By this Git should show that this file was only renamed and not deleted/added.

Reply:
No, I didn't... I renamed it in my IDE. 😔

Reply:
The problem is, I am not able to see the changes in this utility. Could you pls copy the content of the new file somewhere, remove the new file from git, checkout the old file from master, then do a proper git rename and copy the changes back? That should work 🤔

Reply:
I tried this now multiple times, but GitHub ends always up showing them as 'added' and 'deleted'. See:

![image](https://user-images.githubusercontent.com/8037602/30043381-38769dbc-9221-11e7-856f-750a09343f84.png)

Reply:
That is expected, because you remove the new file and re-add the old file. And then you do a proper rename. Just try to push this as extra commit to see if everything looks correct. You can always drop the extra commit.

Reply:
I the first commit I didn't touch these files. I made `git reset HEAD~1` and checked out the deleted files so they're in their old state again. Pasted in the code and deleted the new files. This I commited (but not separate) Then I did the `git mv..` which I then amended to the commit. Now I have two followed commits already. I can squash them but I kept them separate for readability.

Reply:
FYI: This is a missing Github UI feature. First it does not show here that the file was moved, but it was. And second Github UI doesn't support following the history of a moved file (`git log --follow`).

Reply:
FWIW I think this is a Git problem rather than a Github problem - if you rename a file but change a bunch of lines in that file, it is easier for Git to say you removed the old one and added a new one.